lib/stdlib/src/erl_lint.erl

```diff
@@ -2009,13 +2009,19 @@ bit_size_check(Anno, Size, #bittype{type=Type,unit=Unit}, St) ->
     St2 = elemtype_check(Anno, Type, Sz, St),
     {Sz,St2}.
 
+elemtype_check(_Anno, float, 16, St) -> St;
 elemtype_check(_Anno, float, 32, St) -> St;
 elemtype_check(_Anno, float, 64, St) -> St;
 elemtype_check(Anno, float, _Size, St) ->
     add_warning(Anno, {bad_bitsize,"float"}, St);
 elemtype_check(_Anno, _Type, _Size, St) ->  St.
 
 
+%% add_bit_size(Line, ElementSize, BinSize, Build, State) -> {Size,State}.
+%%  Add bits to group size.
+>>>>>>> Support IEEE 754-2008 16-bit floats in bitstrings
```

Comment:
Seems like you have committed part of the merge?

Reply:
GAAAAAAAAAAAAH. Thank you! Fixed.